frontend/src/metabase/dashboard/components/Dashboard.jsx

```diff
@@ -331,7 +337,11 @@ export default class Dashboard extends Component {
                   )}
                 </div>
               </div>
-              <Sidebars {...this.props} isSharing={isSharing} />
+              <Sidebars
+                {...this.props}
+                isSharing={isSharing}
+                onCancel={this.onCancel}
+              />
```

Comment:
Tbh it's still a little perverse to me that I need to thread both the state (`isSharing`) and its mutator (`onCancel`) through, but I can roll with it. I think it'd be more compelling if the data was more complex than a single boolean value.

Reply:
oh, d'oh, good call